requirements-dev.txt

```diff
@@ -5,8 +5,8 @@ flake8==3.7.9
 nose==1.3.7
 splinter>=0.13.0
 -e git+https://github.com/qld-gov-au/ckanext-datarequests@training#egg=ckanext-datarequests
--e git+https://github.com/qld-gov-au/ckanext-data-qld@develop#egg=ckanext-data_qld
+-e git+https://github.com/salsadigitalauorg/ckanext-data-qld@develop#egg=ckanext-data_qld
 -e git+https://github.com/ckan/ckantoolkit@release-0.0.4#egg=ckantoolkit
 -e git+https://github.com/ckan/ckanapi@ckanapi-4.3#egg=ckanapi
 -e git+https://github.com/ckan/ckanext-scheming@release-1.2.0#egg=ckanext-scheming
--e git+https://github.com/qld-gov-au/ckanext-data-qld-theme@develop#egg=ckanext-data_qld_theme
+-e git+https://github.com/salsadigitalauorg/ckanext-data-qld-theme@develop#egg=ckanext-data_qld_theme
```

Comment:
todo, revert back to qld-gov-au once merged.

Reply:
is it possible to run this without our theme as i don't think ckanext-ytp-comments wants to be dependant on our theme for their success.

Reply:
@duttonw Hmm, yes I hear you re: making the `ckanext-ytp-comments` tests independent of the data-qld-theme extension, however it's a little bit of a grey area at the moment. The ability to display comments on a separate page/tab to the dataset details is now configurable, so the decision to display them on a separate tab would occur on a per-site basis. In this case, it is a requirement for the Data.Qld site, so the template for doing so has been implemented within the `ckanext-data-qld-theme` extension, and so the tests are based on the assumption that `show_comments_tab_page` config setting is set to True. I will try to think of a way we could incorporate this and keep it as generic as possible while keeping in mind that we've already demolished the timebox provision for automated testing on this epic... can we proceed with getting this one into QA in the meantime, or does this need to be resolved immediately? Cheers.

Reply:
lets add a ticket to the backlog and move forward then.

Reply:
have merged the theme, please revert back to qld-gov-au version please.